### PR TITLE
Job list

### PR DESCRIPTION
**Pull request by BaltrusS**

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -1,13 +1,16 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
-import { HomePageComponent } from './home-page/home-page.component';
-import { JobRegistrationFormComponent } from './job-registration-form/job-registration-form.component';
+import {HomePageComponent} from './home-page/home-page.component';
```
Comment (matasgraj) [inline src/app/app-routing.module.ts]: Format all files you have changed.

File: src/app/job-list/job-list.component.scss [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+.mat-row {
+    padding: 10px;
+}
+
+.detail-row {
+    padding: 0;
+}
+
+.mat-footer-row,
```
Comment (matasgraj) [inline src/app/job-list/job-list.component.scss:20]: This class duplicates with class at 7th line. Leave one

File: src/app/job-list/job-list.component.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    ]
+})
+export class JobListComponent {
+    dataSource: Job[];
+    columnsToDisplay = ['organisation', 'idea', 'city', 'category', 'date', 'button'];
+    expandedElement: Job | null;
+
+    constructor(private jobService: JobService) {
+        this.jobService.getJobList().subscribe(value => {
+            console.log(value);
```
Comment (matasgraj) [inline src/app/job-list/job-list.component.ts]: Remove `console.log`

**Review by matasgraj — CHANGES_REQUESTED** [review]
All files need to be formatted with Prettier

**Review by matasgraj — APPROVED** [review]

**Review by RokasAlechnavicius — APPROVED** [review]

**Pull request merged**